index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>TankLot</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+</head>
+<nav class="navbar navbar-inverse navbar-fixed-top">
+<a class="navbar-brand" href="#">Tank Depot</a>
+<form class="navbar-form navbar-left" role="search">
```

Comment:
Change this form tag to a div. A form tag is for submitting a form and causes the page to refresh when you press enter.